package.json

```diff
@@ -148,8 +128,55 @@
                 "iiq-dev-accelerator.mode": {
```

Comment:
Also please put your name and email in the contributors list

Reply:
done